cli/replicaset/lua/cconfig/bootstrap_vshard_body.lua

```diff
@@ -1,12 +1,15 @@
+local sharding_roles = require('config'):get().sharding.roles
+if sharding_roles == nil then
```

Comment:
I would suggest to make the check more careful:
```suggestion
local sharding = require('config'):get().sharding
if sharding == nil or sharding.roles == nil then
```